lib/web/desktop.go

```diff
@@ -392,6 +392,33 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 	return trace.NewAggregate(retErrs...)
 }
 
+// wasWebsocketClosedByClient handles the error returned by proxyWebsocketConn
+func handleProxyWebsocketConnErr(err error, log *logrus.Entry) {
```

Comment:
Rename `err` to something else, so it stands out in relation to local errors? `websocketErr` maybe?

Reply:
d9d1d6f87852a508b34da1da7665c013ff785feb